### PR TITLE
show how to run a specific seed file

### PR DESCRIPTION
**Pull request by lmj0011**

### CODE REVIEW

**Comment by lmj0011** [comment]
I assume the `--file` flag is comma delimited

File: database/04-seeds-and-factories.adoc [hunk 1]
```diff
@@ -58,12 +58,18 @@ class DatabaseSeeder {
 module.exports = DatabaseSeeder
 ----
 
-.Running The Seed Command
+.Running The Seed Command - _for all seeds_
```
Comment (thetutlage) [inline database/04-seeds-and-factories.adoc]: The formatting is so different from what we have in the docs till now. Would you mind updating it as defined below

1. Let the heading **Running the Seed Command** as it is
2. After the 1st code example, create a simple paragraph saying. `Run the following command to execute a single seed file`

Comment (lmj0011) [inline database/04-seeds-and-factories.adoc]: this is silly, you could have easily made that edit yourself.

**Comment by thetutlage** [comment]
Thanks a ton 😄

**Pull request merged**